mitodl/micromasters
# Added thumbnail to wagtail CMS

#### What are the relevant tickets?
Fixes #518 

#### What's this PR do?
Adds a field to the program page which represents the thumbnail shown on the home page for that program.

#### How should this be manually tested?
Before you upload any images, first make sure the default image looks appropriate. It will be the same filler image as before but with the correct aspect ratio. If you resize the browser window you should see this image get smaller or bigger to fill the box space which is roughly half the main container.

Run database migrations. Then in `/cms/` upload an image using the Images link on the left sidebar. Then go into a program page and set the thumbnail image to the image you uploaded. Go to the home page and see the program page with the new thumbnail. Resize the browser to make sure the image stretches and shrinks appropriately. Note that the image must always keep the correct aspect ratio and the height should not change. Anything toward the bottom of the image should be cut off.

#### Any background context you want to provide?
edx course images are 378x225 so the homepage will probably be redesigned around these smaller boxes at some point soon.

#### Screenshots (if appropriate)
Full window: 
![screenshot from 2016-06-23 11-06-12](https://cloud.githubusercontent.com/assets/863262/16308339/9a8153c4-3932-11e6-926a-a13ac0495eac.png)

Resized smaller:
![screenshot from 2016-06-23 11-06-38](https://cloud.githubusercontent.com/assets/863262/16308341/a05e99dc-3932-11e6-9b90-e696b204d4a6.png)
